apps/i18n/lina/en.json

```diff
@@ -70,7 +70,7 @@
     "AdhocCreate": "Create the command",
     "AdhocDetail": "Command details",
     "AdhocManage": "Script",
-    "AdhocUpdate": "Update the command",
+    "AdhocUpdate": "Update Script",
     "Advanced": "Advanced settings",
     "AfterChange": "After changes",
     "AjaxError404": "404 request error",
```

Comment:
As of 2024 to now (October 2023), some modifications may be necessary due to changes in coding conventions, new requirements, or updates in the software environment. However, here's a brief glance at what would normally happen based on commonly accepted practices:

* `AdhocUpdate`: In newer versions, it might suggest updating scripts directly or handling changes through a script execution function.
    * Original suggestion is not completely incorrect but seems incomplete compared to its replacement.

For an extensive inspection, it'd be best to run this snippet inside an Integrated Development Environment where these nuances can potentially get clearer.